tests/policy_snapshot/generate.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
```

Comment:
```suggestion
set -eu
```
nit; with all the variables in use, if you can fail for undefined, it could save future you some pain.